kentik_api_library/kentik_api/public/device_label.py

```diff
@@ -21,13 +22,13 @@ class DeviceLabel:
     def __init__(
         self,
         name: str,
-        color: Optional[str] = None,
-        id: Optional[ID] = None,
-        user_id: Optional[ID] = None,
-        company_id: Optional[ID] = None,
-        devices: Optional[List[DeviceItem]] = None,
-        created_date: Optional[str] = None,
-        updated_date: Optional[str] = None,
+        color: str,
+        devices: List[DeviceItem],
+        id: ID,
+        user_id: ID,
+        company_id: ID,
+        created_date: str,
+        updated_date: str,
```

Comment:
I think we also could change type of these to datetime

Reply:
Yes, we can provide time parsing function and update the types across all the classes in public/